delete.raml

```diff
@@ -0,0 +1,41 @@
+#%RAML 0.8
+title: Delete
+version: v0.1.1-20180620
+documentation:
+
+  - title: OCF Copyright
+    content: Copyright (c) 2018 Open Connectivity Foundation, Inc. All rights reserved.
+  - title: OCF License
+    content: !include LICENSE.md
+  - title: OCF Disclaimer
+    content: !include DISCLAIMER.md
+
+
+schemas:
+  - delete-Response: !include schemas/oic.if-delete-response-schema.json
+
+traits:
+  - interface:
+      queryParameters:
+        if:
+          enum: ["oic.if.delete"]
+
+
+
+/DeleteResURI?if=oic.if.delete:
+    description: |
+      Delete interface
+    displayName: delete
+
+    delete:
+        description: |
+          Delete method processing
+        responses:
+          200:
+            body:
+              application/json:
+                schema: delete-response
+                  example: |
+                    {
+                      "href": "/rules/heat-on/fanmode"
```

Comment:
The "href" is supposed to be relative to the parent Collection, right?  I assume that "/rules/heat-on" is the URI of the Collection.  Or maybe the Collection is "/rules".  May want to change this "href" to make it clear that it's relative to the Collection.